src/CMakeLists.txt

```diff
@@ -49,6 +49,19 @@ endif (ENABLE_DASSERT)
 
 message( STATUS "ENABLE_DASSERT : ${ENABLE_DASSERT}")
 
+# Option (To enable/disable ENABLE_TSDB_ADDB mode.)
+option(ENABLE_TSDB_ADDB "Enable ENABLE_TSDB_ADDB mode." ON)
```

Comment:
We want to leave it on by default?

Reply:
The effective default value comes from cortx-posix's build.sh (root of build scripts). 
About the default value, I'm under the impression that addb in cortxfs should remain on by default,. If this is incorrect, pls let me know.

Reply:
I will let @sachinpunadikar take a call on that.Everytime we do a ganesha restart or ganesha is killed, ADDB dumps are generated. These will eventually accumulate and will need to be cleaned up manually.If thats not an issue, we can keep it enabled

Reply:
By default ADDB tracing should be OFF.

Reply:
Thanks, all the configs are now updated to set to off by default. But pls note that though the addb traces in cortxfs are turned off by default, some other components, possible motr is generating addb traces. Hence you will continue to see those add dirs after killing NFS ganesha even though cortxfs's addb traces are turned off. However those traces won't have any addb traces from cortxfs.